src/enums/coopType.ts

```diff
@@ -1,5 +1,5 @@
 export enum CoopType {
-  COOPERATIVE = 'Cooperative Without a Special Purpose',
```

Comment:
Hah, I was wondering if this type of coop was having an existential crisis or something.

Reply:
hahaha... oh man. 
This was the original iteration too! Too bad we couldn't leave it like this 😆